go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/conduitio/conduit-connector-protocol v0.5.0
 	github.com/conduitio/conduit-connector-s3 v0.5.0
 	github.com/conduitio/conduit-connector-sdk v0.8.0
+	github.com/conduitio/conduit-processor-sdk v0.0.0-20231226181810-64784c1029dd
```

Comment:
Needs to be updated once https://github.com/ConduitIO/conduit-processor-sdk/pull/2 and https://github.com/ConduitIO/conduit-processor-sdk/pull/7 are merged.